### ISSUE TITLE
1822. Sign of the Product of an Array

### ISSUE THREAD

**Issue by sayyidkhan** [LeetCode-Feedback/LeetCode-Feedback#12687]
#### Question link:
https://leetcode.com/problems/sign-of-the-product-of-an-array/

#### Your LeetCode username

sayyidkhan

#### Category of the bug
- [x] Question
- [ ] Solution
- [ ] Language
- [ ] Missing Test Cases 

#### Description of the bug

the description is unclear. 
`Let product be the product of all values in the array nums.`
this can mean addition of everything in the array when it should be a multiplication.

#### Expected behavior

need to leave an example:
eg. nums = [-1,-2,-3,-4,3,2,1]
-1 * -2 * -3 * -4 * 3 * 2 * 1

basically just explain that `product` is the sum of all the values in the array by multiplying each and every value in the list.

```
Let product be the sum of all the values in the array by multiplying each and every value in the list.
eg. nums = [-1,-2,-3,-4,3,2,1]
-1 * -2 * -3 * -4 * 3 * 2 * 1 == 144
```

**Comment by LC-Pam** [comment]
Hi there,

Thank you for reaching out to us.

 We've relayed this issue to our team to investigate. 

We will reach back to you after we have the results.

Thank you for your support.

**Comment by LC-Pam** [comment]
Thank you for your time. 

We've used your feedback to check the problem. 

The team decided to reject this feedback after the evaluation. 

Product is * almost always and example is there.

If you have any other questions or feedback, please don't hesitate to let us know! 

We appreciate your support!
